File: day 3/leapYear.py
year = int(input("Enter the year: "))

if year % 4 == 0 & year % 100 == 0:
    print("It's a leap year!")
elif year % 4 == 0 & year % 400 == 0:
    print("It's a leap year!")
else:
    print("Not a leap year.") 

File: day3/leapYear.js
const year = 2000;

year % 4 === 0 && year % 100 === 0
  ? console.log("Leap year.")
  : year % 4 === 0 && year % 400 === 0
  ? console.log("Leap year.")
  : console.log("Not a leap year.");

